packages/core/src/codewhispererChat/editor/context/file/languages.ts

```diff
@@ -117,54 +117,7 @@ export function extractLanguageNameFromFile(file: TextDocument): string | undefi
 export function extractAdditionalLanguageMatchPoliciesFromFile(file: TextDocument): Set<string> {
     const languageId = file.languageId
 
-    if (languageId === undefined) {
-        return new Set<string>()
-    }
-    if (
-        [
-            'yaml',
-            'xsl',
-            'xml',
-            'vue',
-            'tex',
-            'typescript',
-            'swift',
-            'stylus',
-            'sql',
-            'slim',
-            'shaderlab',
-            'sass',
-            'rust',
-            'ruby',
-            'r',
-            'python',
-            'pug',
-            'powershell',
-            'php',
-            'perl',
-            'markdown',
-            'makefile',
-            'lua',
-            'less',
-            'latex',
-            'json',
-            'javascript',
-            'java',
-            'ini',
-            'html',
-            'haml',
-            'handlebars',
-            'groovy',
-            'go',
-            'diff',
-            'css',
-            'c',
-            'coffeescript',
-            'clojure',
-            'bibtex',
-            'abap',
-        ].includes(languageId)
-    ) {
+    if (languageId === undefined || defaultLanguages.includes(languageId)) {
         return new Set<string>()
     }
     switch (languageId) {
```

Comment:
out of scope, but man this switch statement is brutal. Could probably cut the character count in half by using pass-through and not typing `new Set<string>` everywhere.

Reply:
Yeah I thought the same thing lol. These functions are basically just a giant map with two values.